model/ActivityMonitoringService.php

```diff
@@ -131,8 +129,6 @@ public function getData()
             self::STATE_IN_PROGRESS_ASSESSMENTS => $active
         ];
 
-        $deliveryStates = $this->getStatesByDelivery();
```

Comment:
Also I deleted this, because I didn't find where this used, please, check it twice